MovieViewer/Movie.swift

```diff
@@ -8,19 +8,47 @@
 
 import Foundation
 
+var favoritedMovies = [Int]()
```

Comment:
oh, I see what you did - you made this a global variable, now I understand why this works. Move this inside struct movie, add a `static`, and you have something a tiny bit cleaner. Although...I still don't love it. 😫